### ISSUE TITLE
First install on raspberry pi 4 64-bit fails to run download of litecoin-0.21.3

### ISSUE THREAD

**Issue by savaged** [litecoin-project/litecoin#954]
Downloaded the arm version of litecoin-0.21.3 to a fresh install of RaspberryOS 64-bit (Lite). Extracted to the home directory and cannot run litecoind.

Expected to execute litecoind and run a full node.

Actual result after I tried to run litecoind was the following:
`-bash: litecoin-0.21.3/bin/litecoind: cannot execute: required file not found`

Done a bit of searching and the only thing I’ve found is that the file is DOS format, but i don’t believe that.

**Comment by losh11** [comment]
You’re using the aarch64 binary correct?

**Comment by savaged** [comment]
I wasn't; I was using arm.  I've downloaded the aarch64 version and it works right away! Thank you

**Comment by losh11** [comment]
Yes aarch64 is the name they gave to arm64 cpus for some reason, kinda
confusing for the avg person though.
